src/Microsoft.PowerShell.Security/security/CredentialCommands.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.PowerShell.Commands
     /// operations involving security.
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "Credential", DefaultParameterSetName = GetCredentialCommand.credentialSet, HelpUri = "http://go.microsoft.com/fwlink/?LinkID=113311")]
-    [OutputType(typeof(PSCredential), ParameterSetName = new string[] { GetCredentialCommand.credentialSet, GetCredentialCommand.messageSet })]
+    [OutputType(typeof(PSCredential), ParameterSetName = new string[] { GetCredentialCommand.credentialSet, GetCredentialCommand.messageSet, GetCredentialCommand.titleSet, GetCredentialCommand.usernameSet })]
```

Comment:
It feels not necessary to create two more parameter sets.
How about making `Message` not mandatory? It won't be a breaking change to existing scripts.
Then you can add `Title` to the existing `MessageSet` and the `ValidateNotNullOrEmpty` attribute can be added to `Message` and `Title`.